mopidy_gmusic/session.py

```diff
@@ -114,7 +114,7 @@ def get_artist_info(
 
     @endpoint(default=None, require_all_access=True)
     def search_all_access(self, query, max_results=50):
```

Comment:
This method should be renamed to `search` and the `require_all_access` kwarg set to `False`, ref the commit message on the gmusicapi commit that changed this, because all access was no longer required.